nextbike/datapreparation/feature_add.py

```diff
@@ -28,22 +30,33 @@ def additional_feature_creation(p_df_trips):
     p_df_trips['Duration'] = ((p_df_trips['End Time'] - p_df_trips['Start Time']).dt.total_seconds() / 60.0).round(2)
     # TODO: Fix that bad style right here:
     pd.options.mode.chained_assignment = None
-    p_df_trips["month_start"] = p_df_trips["Start Time"].dt.month
-    p_df_trips["day_start"] = p_df_trips["Start Time"].dt.day
-    p_df_trips["hour_start"] = p_df_trips["Start Time"].dt.hour
-    p_df_trips["dayofyear_start"] = p_df_trips["Start Time"].dt.dayofyear
+    p_df_trips["Month_start"] = p_df_trips["Start Time"].dt.month
+    p_df_trips["Day_start"] = p_df_trips["Start Time"].dt.day
```

Comment:
Month und day are the same on start and end - we could just use month & day on start time - like we did on weekend

otherwise we should do the same with weekend and also calculate it for start and end

Reply:
discussed in live session. we leave it like this at first